app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -905,7 +905,7 @@ public function getTopSubmitters($limit = 10, $dateFrom = null, $dateTo = null,
         $chartQuery->applyFilters($q, $filters);
         $chartQuery->applyDateFilters($q, 'date_submitted');
 
-        return $q->execute()->fetchAll();
+        return $q->executeQuery()->fetchAllAssociative();
```

Comment:
```suggestion
        return $q->execute()->fetchAllAssociative();
```
executeQuery is a method of Connection class and this method is not exist in QueryBuilder class.

Reply:
fixed in commit 3eb163cc33db937f70e871a4d07ebc143fd10bb3